predict.py

```diff
@@ -115,11 +116,17 @@ def predict(
             generator=generator,
             num_inference_steps=num_inference_steps,
         )
-        if any(output["nsfw_content_detected"]):
-            raise Exception("NSFW content detected, please try a different prompt")
 
+        samples = [output["sample"][i] for i, nsfw_flag in enumerate(output["nsfw_content_detected"]) if not nsfw_flag]
+
+        if len(samples) == 0:
+            raise Exception(f"NSFW content detected in all outputs, please try a different prompt")
```

Comment:
```suggestion
            raise Exception(f"NSFW content detected. Try running it again, or try a different prompt.")
```

Thinking of a user seeing this when running the web form on Replicate here. "All outputs" doesn't make sense in the default case, and trying again will usually fix because it will give you a different seed. Also fixed a comma splice.